importScripts/sampleRecords/itemData.source.json

```diff
@@ -1,602 +1,547 @@
-  "StatementOfSignificance": "Historic: \r\nSocial:",
-  "Materials": "Silk",
-  "Classification": "dress",
-  "PrimaryClass": "COSTUME",
-  "SecondaryClass": "Female",
-  "TertiaryClass": "dress"
- },
- {
-  "Accession_Full_ID": "P2014.25",
-  "ItemName": "GREETING CARD WITH PAINTING OF CARNARVON DRY RIVER BED",
-  "Summary": "Greeting card with painting titled 'Dry River bed near Carnarvon, W.A.' signed P Berry\r\nPrinted in green inside card 'Dry river bed near Carnarvon, WA' By Paul Berry. /Paul, paralysed in the arms by poliomyelitis, paints with his foot.\r\nWith Best Wishes/for Christmas/and the New Year\r\nHandwritten message in black ink",
-  "Creator": "painting by Paul Berry for Paraplegic Association of WA",
-  "ComparativeCriteria": "representativeness",
-  "PhysicalCondition": "2",
-  "Significance_Representativeness": "1",
-  "StatementOfSignificance": "Representativeness: Greeting card with painting of Carnarvon done for Paraplegic Association of WA",
-  "Materials": "paper, printing ink and writing ink",
-  "Dimensions": "10 cm x 5 cm",
-  "Classification": "dry river bed",
-  "PrimaryClass": "PLACES",
-  "SecondaryClass": "Carnarvon",
-  "TertiaryClass": "dry river bed"
- },
- {
-  "Accession_Full_ID": "P2014.807",
-  "ItemName": "NURSE PEARL ASHWIN MEEKATHARRA HOSPITAL",
-  "Summary": "Post card. Nurse Pearl Ashwin in nursing uniform  Meekatharra Hospital\r\n\r\nPrinted on reverse: Image: Nurse Pearl Ashwin, Meekatharra Hospital/State Library of Western Australia, BA368/278/My Story: Pearl Ashwin was one of the first Aboriginal women in Meekatharra to become a registered nurse and is pictured here working at Meekatharra Hospital.\r\n\r\nStorylines is a State Library initiative to explore with the community the last 125 years of Aboriginal history through our heritage collections.",
-  "ComparativeCriteria": "interpretive",
-  "Significance_Interpretive": "1",
-  "PhysicalCondition": "0",
-  "StatementOfSignificance": "Interpretive: Photograph of Nurse Pearl Ashwin one of the first Aboriginal women in Meekatharra to become a registered nurse",
-  "Materials": "photographic paper and printing ink",
-  "Dimensions": "W:10 x H 14",
-  "Classification": "ashwin pearl",
-  "PrimaryClass": "PERSON",
-  "SecondaryClass": "Individual",
-  "TertiaryClass": "ashwin pearl"
- },
- {
-  "Accession_Full_ID": "P2015.140",
-  "ItemName": "TELECOM HISTORICAL PICTURES",
-  "Summary": "Telecom Historical Pictures - PMG Light Truck",
-  "ItemDates": "1940s",
-  "EarliestYear": "1940",
-  "LatestYear": "1949",
-  "SignificanceCriteria": "historic",
-  "Significance_Historic": "1",
-  "ComparativeCriteria": "representativeness",
-  "PhysicalCondition": "0",
-  "Significance_Representativeness": "1",
-  "StatementOfSignificance": "Historic: \r\nRepresentativeness:",
-  "Materials": "35 mm negative",
-  "Classification": "pmg light truck",
-  "PrimaryClass": "COMMUNICATION",
-  "SecondaryClass": "Postal",
-  "TertiaryClass": "pmg light truck"
- }
+[
```

Comment:
This file has different whitespace due to an update of my IDE. if you want to see the actual change append `?w=1` to the URL (this hides any whitespace diffs).